e2e/cypress/integration/channel/send_message_via_profile_popover_spec.js

```diff
@@ -68,21 +68,25 @@ function verifyDMChannelViaSendMessage(postId, profileSelector, user) {
     cy.visit('/ad-1/channels/town-square');
 
     // # Visit post thread on RHS and verify that RHS is opened
+    cy.wait(TIMEOUTS.TINY);
     cy.clickPostCommentIcon(postId);
     cy.get('#rhsContainer').should('be.visible');
 
     // # Open profile popover with the given selector
+    cy.wait(TIMEOUTS.TINY);
     cy.get(`#rhsPost_${postId}`).should('be.visible').within(() => {
```

Comment:
I suggest to use `timeout` option in getting element so it will resume as soon as the element passed the assertion, and not wait on fixed period of time.

Reply:
@saturninoabril I tried that but it still failed on me. I think the app components gets refreshed in the background somehow and the component gets detached from DOM. But again, my local seems kind of different given the other 2 PRs :(

Reply:
Thanks, then I'm good with it.